doc/data_workflow.rst

```diff
@@ -0,0 +1,100 @@
+..
+  SPDX-FileCopyrightText: 2021 The PyPSA meets Earth authors
+
+  SPDX-License-Identifier: CC-BY-4.0
+
+.. _data_workflow:
+
+##########################################
+Data used by the model
+##########################################
+
+PyPSA-Earth has the flexible scope when a modeling domain can be selected from the whole Earth to any subregion. The modeling workflow includes data management workflow which is focused on open data collection, modification, prediction and validation. The developed data processing approaches aimed to provide model-ready data with temporal and spatial resolution as high as it's needed by a specific modeling task.
+
+.. _data_management_strategy:
+
+Data management strategy
+===================================
+
+Here we'll look into architecture of the data workflow while practical hand-ons are given in the Tutorial section.
+
+1. Grid topology data
+===================================
+
+OpenStreetMap `OSM <https://www.openstreetmap.org/>`_ data are used to build power grid topology model. OSM is the biggest crowd-sourced collection of geographic information, which is daily updated and includes geolocation references.
+
+.. image:: img/africa_osm_map.jpeg
+    :height: 150 px
+    :align: center
+
+The raw OSM data are being loaded when running the `download_osm_data` rule and stored in the folder `pypsa-earth/data/osm/{region}/pbf/`. Here `region` denotes a continent, e.g. Africa, or a macro region, e.g. Central America, where the countries of interest belong. The pbf-files contain the entire OSM data for the country; the specific network information related to generators, substations, lines and cables are extracted, cleaned and writen as a geojsons in the folder `pypsa-earth/data/osm/{region}/Elements/`. All network data (generators, substations, lines and cables) for each country are stored as geojson files.
+
+The cleaned OSM network data that are the output of the `osm_data_cleaning` rule, which process the raw OSM data to obtain cleaned datasets of all the network assets, namely generators, substations, lines and cables. These data are stored in `/resources/osm/` folder.
+
+2. Climate data
+===================================
+
+The climate data processing is provided by `atlite <https://atlite.readthedocs.io/en/latest/>`_ package. It extracts all the required whether and climate data to generate the time series of renewable potential by `generate_renewable_profiles` rule. The main data source on climate variables is `ERA5 reanalysis <https://rmets.onlinelibrary.wiley.com/doi/10.1002/qj.3803>`_.
+
+.. image:: img/era5.png
+    :height: 150 px
+    :align: center
+
+3. General data
+===================================
+
+There are a number datasets applied in PyPSA-Earth to build a realistic model. Original datasets are stored in the `pypsa-earth/data/` folder. Currently we are using the following resources.
+
+Environmental
+------------------------------------
```

Comment:
If we use the "-----" for the 1. 2. 3. titles, then there would be the need to revise this title.
In this case, we could use also nested lists something like:
- list 1
- list 2
  - 2.1
  - 2.2
 
for example